__tests__/push/__snapshots__/index.test.ts.snap

```diff
@@ -4,7 +4,6 @@ exports[`Push API handle streamed response 1`] = `
 "> preparing all monitors
 > creating all monitors
 > journey 1
-> journey 2
```

Comment:
any reason why we are not getting updates of journey 2?

Reply:
this is just a test unit behaviour , since api is called only once, in the test first response is logged.

Reply:
Can you change tests to account for multiple response chunks from the streaming endpoint. Thanks.

Reply:
actually in case of multiple jounrey it still logs one response, but yeah the text can be improved , let me do that

Reply:
updated the test